signatures/rego/anti_debugging_ptraceme.rego

```diff
@@ -19,6 +19,23 @@ tracee_selected_events[eventSelector] {
 	}
 }
 
+filters := [
+	{
+		"field": "event",
+		"operator": 0, #Equal
```

Comment:
I would recommend adding an alias to that in the helpers.rego so we could use `equal` instead of `0`

Reply:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/22661609/174562832-51daab9a-bdbd-40f6-af1f-d38126bd1543.png">
WDYT?

Reply:
sounds good although I would even just call it equal but it's really a preference at this point.